src/app/Fake.Sql.DacPac/Sql.DacPac.fs

```diff
@@ -21,8 +21,6 @@ module DacPac =
 
     /// Configuration arguments for DacPac deploy
     type DeployDbArgs = {
-        /// The path to SqlPackage.exe.
-        SqlPackageToolPath : string
```

Comment:
This is basically a breaking change, should we do that, or just leave the field and add a warning if it is set?

Reply:
The latter is probably the better idea. I’ll add back this field.